lexicon/conversion.py

```diff
@@ -140,6 +140,118 @@ def run(self):
             lexicon_file.write(ET.tostring(root, "unicode"))
 
 
+class LexiconFromTextFileJob(Job):
+    """
+    Create a bliss lexicon from space or tab separated files.
+
+    As the splitting is taken from RASR and not fully tested,
+    it might not work in all cases so do not use this job
+    without checking the output manually on new lexicas.
+    """
+
+    def __init__(self, text_file, add_unknown, add_noise, compressed=True):
+        """
+
+        :param Path text_file:
+        :param bool add_unknown: add [UNKNOWN] lemma
+        :param bool add_noise: add [NOISE] lemma
```

Comment:
There could also be other special lemmas, e.g., [LAUGHTER] or [VOCALIZEDNOISE]. We could have a list as input instead where you can specify which special lemmas to use.